packages/esm-patient-orders-app/src/lab-results/lab-results-form.component.tsx

```diff
@@ -221,6 +381,27 @@ const LabResultsForm: React.FC<LabResultsFormProps> = ({
               ) : (
                 <InlineLoading description={t('loadingInitialValues', 'Loading initial values') + '...'} />
               )}
+              <div className={styles.commentsSection}>
+                <Controller
+                  control={control}
+                  name="fulfillerComment"
+                  render={({ field, fieldState: { error } }) => (
+                    <TextArea
+                      {...field}
+                      id="fulfillerComment"
+                      labelText={t('comments', 'Comments')}
+                      placeholder={t('addCommentsOptional', 'Add comments (optional)...')}
+                      helperText={`${t(
+                        'commentsHelperText',
+                        'Enter any additional notes or observations about the test results',
+                      )} (${commentValue.length}/500)`}
+                      invalid={Boolean(error?.message)}
+                      invalidText={error?.message}
```

Comment:
```suggestion
                      invalid={Boolean(errors.fulfillerComment)}
                      invalidText={errors.fulfillerComment?.message}
```

This seems cleaner to me.